dist/PublicLab.Editor.js

```diff
@@ -23358,13 +23358,15 @@ module.exports = function initTables(_module, wysiwyg) {
     $("#tableRows").text( Number($("#tableRows").text()) + 1 );
   });
   $(document).on('click', '#decRows', function() {
-    $("#tableRows").text( Number($("#tableRows").text()) - 1 );
+    const numOfRows = Number($("#tableRows").text());
+    if (numOfRows > 1) $("#tableRows").text( numOfRows - 1 );
   });
   $(document).on('click', '#incCols', function() {
     $("#tableCols").text( Number($("#tableCols").text()) + 1 );
   });
   $(document).on('click', '#decCols', function() {
-    $("#tableCols").text( Number($("#tableCols").text()) - 1 );
+    const numOfCols = Number($("#tableCols").text());
+    if (numOfCols > 1) $("#tableCols").text( numOfCols - 1 );
```

Comment:
Hi @NARUDESIGNS, this file still has those changes.

Reply:
Oops! My bad. 
I should push the dist/PublicLab.Editor.js file as well. 
I think those changes are gone in my local env.

Thank you @TildaDares